server/app/models/ProgramModel.java

```diff
@@ -211,7 +212,7 @@ public ProgramModel(
     this.statusDefinitions = new StatusDefinitions();
     this.versions.add(associatedVersion);
     this.programType = programType;
-    this.eligibilityIsGating = true;
+    this.eligibilityIsGating = eligibilityIsGating;
```

Comment:
Yep, it was defaulted to always be true! It could changed to false if you used `ProgramDefinition.Builder.setEligibilityIsGating(false)` and then built a new `ProgramModel` from that `ProgramDefinition`.

Reply:
Got it. Thanks!